api/client/client.go

```diff
@@ -45,11 +44,20 @@ type Remote interface {
 func NewServer(addr string) Remote {
 	addrs := strings.Split(addr, ",")
 
+	for i := range addrs {
```

Comment:
I think one limitation of net/url is that a url parsed in the format of http://xxxx.com will be parsed as "scheme://[userinfo@]host/path[?query][#fragment]" , where the url.Scheme part will be either http/https.
If the url is without a protocol, ex.127.0.0.1:8888, net/url will parse the url as "scheme:opaque[?query][#fragment]" where url.Scheme is recognized as 127.0.0.1. 
So I think a little string concatenation for url without http/https define here is necessary?